<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetContacts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>211</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>СПИСОК КОНТАКТОВ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenChat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>65</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Общий
 канал</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddContact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>65</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить
 контакт</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditAddContact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>    ----       Информация     -----</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelContact">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>90</y>
      <width>65</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить
 контакт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>210</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавление (удаление) контакта</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>233</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
